yorkie/metrics/metrics_server.go

```diff
@@ -0,0 +1,58 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/yorkie-team/yorkie/pkg/log"
+	"github.com/yorkie-team/yorkie/pkg/version"
+)
+
+type Config struct {
```

Comment:
I missed a lot too, but could you please comment on the exported methods or exported type? I will add comments in future work.
https://goreportcard.com/report/github.com/yorkie-team/yorkie

Reply:
OK. I'll do that.